packages/terra-search-field/src/SearchField.jsx

```diff
@@ -172,6 +174,7 @@ class SearchField extends React.Component {
 
   handleKeyDown(event) {
     if (event.nativeEvent.keyCode === KeyCode.KEY_RETURN) {
+      this.searchBtnRef.focus();
```

Comment:
Thanks, this teaches me the value of using refs more clearly than the doc.

Reply:
`this.searchBtnRef.focus();` this will cause an error since the dom node of button will be available on `current` attribute of ref ( as mentioned [here](https://reactjs.org/docs/refs-and-the-dom.html#accessing-refs)). Calling focus() method directly on ref object will throw `Err: focus() is not a function`.

Reply:
Thanks for the explanation!
Updated here: f4db4098cfb69d72c9f50e870151647eae859479